### PR TITLE
Dw 2884 - timestamp, version, unitOfWorkId, traceId in message.

### PR DESCRIPTION
**Pull request by DanielChicot**

### CODE REVIEW

File: src/main/kotlin/app/batch/HbaseClient.kt [hunk 1]
```diff
@@ -49,5 +49,4 @@ open class HbaseClient(
         }
     }
 
-    fun close() = connection.close()
```
Comment (mark-r-m) [inline src/main/kotlin/app/batch/HbaseClient.kt:52]: Is that really superfluous?

File: src/test/kotlin/app/batch/MessageProducerTest.kt [hunk 2]
```diff
@@ -77,6 +93,20 @@ class MessageProducerTest {
         assertEquals(expectedObject, actual)
     }
 
+    @Test
+    fun testTwoObjectsGetSameGuid() =
+        assertEquals(unitOfWorkId(validJsonOne()),unitOfWorkId(validJsonTwo()))
```
Comment (mark-r-m) [inline src/test/kotlin/app/batch/MessageProducerTest.kt]: Needs formatter applied please

File: src/main/kotlin/app/batch/HbaseClient.kt (hunk 1 above)
Comment (DanielChicot) [reply src/main/kotlin/app/batch/HbaseClient.kt:52]: Hmm that was probably an IntelliJ code inspection change. I'll put a call to it after the pipeline has run though.

File: src/test/kotlin/app/batch/MessageProducerTest.kt (hunk 2 above)
Comment (mark-r-m) [reply src/test/kotlin/app/batch/MessageProducerTest.kt]: You could add formatting on save ;-)

Comment (DanielChicot) [reply src/test/kotlin/app/batch/MessageProducerTest.kt]: sorted.

**Review by mark-r-m — APPROVED** [review]

**Pull request merged**